dxskytap/connect.py

```diff
@@ -31,46 +31,62 @@
 
 Exceptions:
  - SkytapException
- - TimeoutException
  - NoResponseException
 '''
 
+import urllib3.contrib.pyopenssl
+urllib3.contrib.pyopenssl.inject_into_urllib3()
```

Comment:
Because the default backend for urllib3 has to be compiled in. Meaning if the upgrade openssl, in order to see the changes in python you have to recompile and reinstall python core. This is an alternative backend for urllib3 that uses the pyopenssl library. pyopenssl is an alternative library to the core ssl library that provides better control and integration with openssl.